cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -295,5 +295,9 @@ type AuditPolicyConfiguration struct {
 	LogDir string `json:"logDir"`
 	// LogMaxAge is the number of days logs will be stored for. 0 indicates forever.
 	LogMaxAge *int32 `json:"logMaxAge,omitempty"`
+	// WebhookConfigPath is the local path to webhook policy.
+	WebhookConfigPath string `json:"webhookConfigPath"`
```

Comment:
Please add `,omitempty`, so the output of the struct serialization will be cleaner